### ISSUE TITLE
[A11yInspect] Fail: 2.4.3 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#14]
## ❌ Accessibility Fail

**Rule:** 2.4.3 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 1, **Column:** 1

### Description
Incorrect nesting of interactive HTML elements (Major)

### Context
```html
<audio src="/audio.mp3">

        dynamic-content

        dynamic-content
        <a onclick="dynamic-content">Click here</a>

        dynamic-content
        <a href="/more">Read more</a>

        d
```

### Selector
`html > body > div > div > video > audio`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
